$OpenBSD$

change Qt version 5.10 -> 5.9
change LLVM version to look up to 6

Index: rpcs3/CMakeLists.txt
--- rpcs3/CMakeLists.txt.orig
+++ rpcs3/CMakeLists.txt
@@ -6,12 +6,12 @@ set(CMAKE_CXX_STANDARD 14)
 include(CheckCXXCompilerFlag)
 
 # Qt section
-find_package(Qt5 5.10 COMPONENTS Widgets Network)
+find_package(Qt5 5.9 COMPONENTS Widgets Network)
 if(WIN32)
 	find_package(Qt5 5.10 COMPONENTS WinExtras REQUIRED)
 	set(RPCS3_QT_LIBS Qt5::Widgets Qt5::WinExtras Qt5::Network)
 else()
-	find_package(Qt5 5.10 COMPONENTS DBus Gui)
+	find_package(Qt5 5.9 COMPONENTS DBus Gui)
 	if(Qt5DBus_FOUND)
 		set(RPCS3_QT_LIBS Qt5::Widgets Qt5::DBus Qt5::Network)
 		add_definitions(-DHAVE_QTDBUS)
@@ -23,22 +23,6 @@ endif()
 
 # Let's make sure we have Qt before we continue
 if(NOT Qt5Widgets_FOUND)
-	if(Qt5Widgets_VERSION VERSION_LESS 5.10.0)
-		message("Minimum supported Qt5 version is 5.10.0! You have version ${Qt5Widgets_VERSION} installed, please upgrade!")
-		if("${CMAKE_SYSTEM}" MATCHES "Linux")
-			message(FATAL_ERROR "Most distros do not provide an up-to-date version of Qt.
-If you're on Ubuntu or Linux Mint, there are PPAs you can use to install an up-to-date qt5 version.
-        https://launchpad.net/~beineri/+archive/ubuntu/opt-qt-5.10.1-xenial
-        https://launchpad.net/~beineri/+archive/ubuntu/opt-qt-5.10.1-trusty
-just make sure to run
-    source /opt/qt510/bin/qt510-env.sh
-before re-running cmake")
-		elseif(WIN32)
-			message(FATAL_ERROR "You can download the latest version of Qt5 here: https://www.qt.io/download-open-source/")
-		else()
-			message(FATAL_ERROR "Look online for instructions on installing an up-to-date Qt5 on ${CMAKE_SYSTEM}.")
-		endif()
-	endif()
 
 	message("CMake was unable to find Qt5!")
 	if(WIN32)
@@ -173,7 +157,7 @@ set(CMAKE_MODULE_PATH "${RPCS3_SRC_DIR}/cmake_modules"
 find_package(OpenGL REQUIRED)
 find_package(OpenAL REQUIRED)
 if(NOT WITHOUT_LLVM)
-	find_package(LLVM 4.0 CONFIG)
+	find_package(LLVM 6.0 CONFIG)
 	if(NOT LLVM_FOUND)
 		message("System LLVM was not found, LLVM will be built from the submodule.")
 
